contracts/SportMarkets/LiquidityPool/SportAMMLiquidityPool.sol

```diff
@@ -0,0 +1,678 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts-upgradeable/token/ERC20/utils/SafeERC20Upgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import "@openzeppelin/contracts-upgradeable/security/PausableUpgradeable.sol";
+
+import "../../utils/proxy/solidity-0.8.0/ProxyReentrancyGuard.sol";
+import "../../utils/proxy/solidity-0.8.0/ProxyOwned.sol";
+import "@openzeppelin/contracts-4.4.1/proxy/Clones.sol";
+
+import "../../interfaces/ISportsAMM.sol";
+import "../../interfaces/ISportPositionalMarket.sol";
+import "../../interfaces/IStakingThales.sol";
+
+import "./SportAMMLiquidityPoolRound.sol";
+
+contract SportAMMLiquidityPool is Initializable, ProxyOwned, PausableUpgradeable, ProxyReentrancyGuard {
+    /* ========== LIBRARIES ========== */
+    using SafeERC20Upgradeable for IERC20Upgradeable;
+
+    struct InitParams {
+        address _owner;
+        ISportsAMM _sportsAmm;
+        IERC20Upgradeable _sUSD;
+        uint _roundLength;
+        uint _maxAllowedDeposit;
+        uint _minDepositAmount;
+        uint _maxAllowedUsers;
+    }
+
+    /* ========== CONSTANTS ========== */
+    uint private constant HUNDRED = 1e20;
+    uint private constant ONE = 1e18;
+
+    /* ========== STATE VARIABLES ========== */
+
+    ISportsAMM public sportsAMM;
+    IERC20Upgradeable public sUSD;
```

Comment:
we are only using sUSD? and on ARBI it is USDC right? 1e18 both? if not that case was covered right?

Reply:
I think there isnt much work to support USDC, I'll try to implement it

Reply:
Added USDC support